gamemode/core/meta/sh_inventory.lua

```diff
@@ -373,13 +373,10 @@ if (SERVER) then
 				for _ = 1, quantity do
 					self:Add(uniqueID, 1, data)
 				end
-
-				return
```

Comment:
The return shouldn't be omitted as it's needed to break out of the function early. It would be better if you checked the return values for each call to `self:Add` and return inside the loop if there is an error - otherwise we can return true after the loop completes successfully.